src/liballoc/collections/vec_deque.rs

```diff
@@ -2209,6 +2267,16 @@ impl<'a, T> Iterator for Iter<'a, T> {
         final_res
     }
 
+    fn nth(&mut self, n: usize) -> Option<Self::Item> {
+        if n >= count(self.tail, self.head, self.ring.len()) {
+            self.tail = self.head;
+            self.next()
```

Comment:
I'd prefer to just put `None` here to simplify. What's the reason to go through `next`, does it have maintainability benefits (I don't really see it).

Reply:
If we go through next - we can factor it out of the loop, though.

Reply:
Yes, I think there's no reason not to use `None`. Is it okay if I make the change and squash it into the original nth commit, or is that bad form?

Reply:
Sure, that's fine. It's great to not squash in bigger updates, but now you've even mentioned what it is, so it's only more efficent to squash it in.